index.html

```diff
@@ -405,48 +405,27 @@ <h3>Terminology and Notation</h3>
+            <p class=syntaxExample><span class=lnum>8 </span><span class=markup>&nbsp;&nbsp;&lt;<span class="vocabulary">p</span>&gt;</span><span class="shakespeare">What<span class="markup">&amp;#x2019;</span>s in a name? That which we call a rose by any other name would smell as sweet.</span><span class="markup">&lt;/<span class="vocabulary">p</span>&gt;</p>
+            <p class=syntaxExample><span class=lnum>9 </span>&lt;/<span class="vocabulary">body</span>&gt;</p>
+            <p class=syntaxExample><span class=lnum>10 </span>&lt;/<span class="vocabulary">html</span>&gt;</span> </p>
           </div>
           <ul style="text-align:left">
-          <li>Everything inside the black rectangle (that is, in this HTML file)
-            is part of the resource.</li>
-          <li><a>Syntactic content</a> is shown in a <span class="markup">monospaced font</span>.</li>
-          <li><a>Natural language content</a> is shown in a <span class="shakespeare">bold
-              blue font with a gray background</span>.</li>
-          <li>User values are shown in <span class="userValue">italics</span>.</li>
-          <li><a>Vocabulary</a> is shown with <span class="vocabulary">red underlining</span>.</li>
-          <li>All of the text above (all text in a text file) makes up a
-            resource. It's possible that a given resource will contain no
-            natural language content at all (consider an HTML document
-            consisting of four empty <code>div</code> elements styled to be
-            orange rectangles). It's also possible that a resource will contain
-            <em>no</em> syntactic content and consist solely of natural language content:
-            for example, a plain text file with a soliloquy from <cite>Hamlet</cite>
-            in it. Notice too that the HTML entity <code>&amp;#x2019;</code>
-            appears in the natural language content and belongs to both the
-          natural language content and the syntactic content in this resource.</li>
+          <li>Everything inside the black rectangle (that is, in this HTML file) is part of the <a>resource</a>.</li>
+          <li><a>Syntactic content</a> in this case includes all of the HTML markup. There are only two strings that are <strong>not</strong> part of the syntactic content: the word <em>"Shakespeare"</em> on line 4 and the sentence <em>"What&#x2019;s in a name? That which we call a rose by any other name would smell as sweet."</em> on line 8. (The HTML entity <q><kbd>&amp;#x2019;</kbd></q> embedded in the sentence on line 8 <em>is</em> part of the syntactic content.)</li>
```

Comment:
Actually, including the alt attribute value (which you call out below), i think there are 3 string that are not part of the syntactic content, not 2.

Reply:
The `alt` attribute value is part of the syntactic content. It is also natural language content. The two strings I call out as not being syntactic are the only part of the resource that aren't part of the HTML syntax.

Reply:
I'm not so sure about that (and never was really convinced).  Sure, the attribute itself is part of the syntax, but whether a string is surrounded by "..." or <p>...</p> seems pretty much the same to me.